envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua

```diff
@@ -90,8 +95,17 @@ function log_request(lua_metadata, handle)
     local xff_header = lua_metadata["request.info.xff_header"] or ""
     local source_ip = string.match(xff_header, '[^,]+$') or ""
     local request_id = lua_metadata["request.info.request_id"] or ""
-    local statusCode = handle:headers():get(":status") or "0"
-    handle:logInfo("\nINCOMING_PERMISSIONS { \"method\": \""..method.."\", \"path\": \""..path.."\", \"clientIp\": \""..source_ip.."\", \"clientName\": \""..escape(client_name).."\", \"trustedClient\": "..tostring(trusted_client)..", \"clientAllowedToAllEndpoints\": "..tostring(allowed_client)..", \"protocol\": \""..protocol.."\", \"requestId\": \""..escape(request_id).."\", \"statusCode\": "..statusCode.." }")
+    local status_code = handle:headers():get(":status") or "0"
+    handle:logInfo("\nINCOMING_PERMISSIONS { \"method\": \""..method..
+        "\", \"path\": \""..path..
+        "\", \"clientIp\": \""..source_ip..
+        "\", \"clientName\": \""..escape(client_name)..
+        "\", \"trustedClient\": "..tostring(trusted_client)..
+        ", \"clientAllowedToAllEndpoints\": "..tostring(allowed_client)..
+        ", \"protocol\": \""..protocol..
+        "\", \"requestId\": \""..escape(request_id)..
+        "\", \"statusCode\": "..status_code..
+        ", \"rbacAction\": \""..rbac_action.."\" }")
```

Comment:
Will it be incorrectly `shadow_denied` for allowed client?

Reply:
Allowed clients are not logged at all so we won't reach `log_request` so this won't happen.

Reply:
Are you sure? I think when `clientAllowedToAllEndpoints: true` then `rbacAction: shadow_denied`, even if request is not shadow denied

Reply:
as mentioned here: https://github.com/allegro/envoy-control/pull/243#discussion_r563514380 allowed client is "shadow_denied" so this will be correct.

Reply:
Ok. Now it makes sense with this simplified logic (logging only shadow_denied request). I think @lukidzi should look at this to confirm that we don't need to log allowed_client requests when they are not shadow_denied, because it can't happen.